frontend/src/screens/ProductEditScreen.js

```diff
@@ -7,7 +7,7 @@ import Message from '../components/Message'
 import Loader from '../components/Loader'
 import FormContainer from '../components/FormContainer'
 import { listProductDetails, updateProduct } from '../actions/productActions'
-import { PRODUCT_UPDATE_RESET } from '../constants/productConstants'
+import { PRODUCT_UPDATE_RESET, PRODUCT_UPDATE_RESET } from '../constants/productConstants'
```

Comment:
typo!
should be: 
import { PRODUCT_UPDATE_RESET, **PRODUCT_DETAILS_RESET** } from '../constants/productConstants'

Reply:
thanks for this fix. please clone the latest repo and run it locally. I tested this scenario and could not find any issue.